workflow-job

```diff
@@ -95,7 +95,7 @@ public class CommitStatus implements Serializable {
 }
 
 public class Images implements Serializable {
-    def cassandra = 'cassandra'
+    def cassandra = 'cassandra:2.1.9'
```

Comment:
I think we have to stay consistent, in the James documentation (README in james-project) we are giving the command with the `2.2.3` release.